go/vt/sqlparser/sql.y

```diff
@@ -193,6 +193,7 @@ func skipToEnd(yylex interface{}) {
 %token <bytes> CONVERT CAST
 %token <bytes> SUBSTR SUBSTRING
 %token <bytes> GROUP_CONCAT SEPARATOR
+%token <bytes> TIMESTAMPADD TIMESTAMPDIFF
```

Comment:
Oh right, not sure how I forgot that.

Reply:
Still waiting on this. I can merge this once you've added them.

Reply:
Sorry, I was OoO. I had to put it in reserved_keywords to avoid reduce/shift conflicts:
```
289: shift/reduce conflict (shift 54(0), red'n 822(0)) on '('
state 289
        function_call_nonkeyword:  TIMESTAMPADD.openb sql_id ',' value_expression ',' value_expression closeb 
        non_reserved_keyword:  TIMESTAMPADD.    (822)

        '('  shift 54
        .  reduce 822 (src line 3359)

        openb  goto 564

290: shift/reduce conflict (shift 54(0), red'n 823(0)) on '('
state 290
        function_call_nonkeyword:  TIMESTAMPDIFF.openb sql_id ',' value_expression ',' value_expression closeb 
        non_reserved_keyword:  TIMESTAMPDIFF.    (823)

        '('  shift 54
        .  reduce 823 (src line 3360)

        openb  goto 565
```

I don't pretend to understand it, but for example CURRENT_TIMESTAMP is also in (`function_call_nonkeyword` and) `reserved_keyword`.